docs/Running-behind-apache.md

```diff
@@ -230,6 +261,14 @@ You can also use a unix domain socket. This will also provide a faster route bet
 [web]
     bind to = unix:/tmp/netdata.sock
 ```
+
+Apache 2.4.24+ can not read from /tmp so create your socket in /var/run/netdata
```

Comment:
Apache 2.4.24+ can not read from `/tmp` so create your socket in `/var/run/netdata`

Reply:
I think I've added to the pull request properly. Let me know if I need to do anything else.